src/NHibernate/Dialect/Oracle12cDialect.cs

```diff
@@ -38,5 +38,9 @@ public override SqlString GetLimitString(SqlString querySqlString, SqlString off
 
 			return result.ToSqlString();
 		}
+
+		// 128 since 12.2. https://stackoverflow.com/a/756569/1178314
+		/// <inheritdoc />
+		public override int MaxAliasLength => 128;
```

Comment:
OK, our own CI is 12.1, and fails due to that, should have stayed on the safe side here.